cloud

```diff
@@ -702,23 +717,28 @@ function scheduler_job_withdraw() {
 }
 
 # Deploy to scheduler
-# Args: project prefix trigger_topic
+# Args: project prefix load_queue_trigger_topic pick_notifications_trigger_topic
 function scheduler_deploy() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    declare -r trigger_topic="$1"; shift
+    declare -r load_queue_trigger_topic="$1"; shift
+    declare -r pick_notifications_trigger_topic="$1"; shift
     # Deploy the jobs
-    scheduler_job_pubsub_deploy "$project" "${prefix}trigger" \
-                                "$trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}load_queue_trigger" \
+                                "$load_queue_trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}pick_notifications_trigger" \
+                            "$pick_notifications_trigger_topic" '30 * * * *' '{}'
 }
 
 # Withdraw from the scheduler
 # Args: project prefix
 function scheduler_withdraw() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    scheduler_job_withdraw "$project" "${prefix}trigger" \
-                           "$trigger_topic" '* * * * *' '{}'
+    scheduler_job_withdraw "$project" "${prefix}load_queue_trigger" \
+                           "$load_queue_trigger_topic" '* * * * *' '{}'
+    scheduler_job_withdraw "$project" "${prefix}pick_notifications_trigger" \
+                       "$pick_notifications_trigger_topic" '30 * * * *' '{}'
```

Comment:
Hey, I just noticed, we don't need the last two arguments passed to `scheduler_job_withdraw`. Also, this function was trying to pull the value of `$trigger_topic` out of nowhere - it wasn't passed one! This was only working due to the really harmful way variable scope works in shell: variables defined in the caller are seen in the called function, so code like that could work called from one place (which has the variable), but not the other (which hasn't), one of the things I dislike most about shell. Both my faults.

The proper way to deal with this would be to have a commit adding the explicit `$trigger_topic` argument first, then a commit removing the extra argument to `scheduler_job_withdraw`, and then the commit adding the new feature (with a new argument for the notification-picking topic, and without the extra arguments to `scheduler_job_withdraw`). However, before you could do that (without going crazy trying to juggle all your commits) you need to transition to always having the final commits in your branch, instead of your work commits. I'll try to explain on Slack.